scripts/build-installer-win64.bat

```diff
@@ -24,6 +31,11 @@ set DLLS_URL=https://s3.eu-central-1.amazonaws.com/cardano-sl-testing/DLLs.zip
 @echo ..with LibreSSL version:    %LIBRESSL_VERSION%
 @echo .
 
+@if not [%FASTER_REBUILD%]==[] (@echo WARNING: FASTER_REBUILD set, skipping to frontend packaging       
```

Comment:
FAST vs FASTER is ambigious, let's rather call them SKIP_TO_FRONTEND_BUILD and SKIP_TO_INSTALLER_BUILD